packages/bootstrap/scss/drawer/_variables.scss

```diff
@@ -8,7 +8,7 @@ $drawer-font-size: $font-size !default;
 $drawer-line-height: $line-height !default;
 $drawer-content-padding-x: $padding-x !default;
 $drawer-content-padding-y: $padding-y !default;
-$drawer-item-level-padding-x: k-map-get( $spacing, 4 ) !default;
+$drawer-item-level-padding-x: $drawer-content-padding-x !default;
```

Comment:
Do we want to modify the hierarchy items padding when changing the padding of the drawer content ? I think no, thus, better use `$padding-x` here.